src/components/font_size_editor/font_size_editor.ts

```diff
@@ -31,6 +31,14 @@ css/* scss */ `
     input::-webkit-inner-spin-button {
       -webkit-appearance: none;
     }
+
```

Comment:
Maybe a bit less padding for menu w/ selected items ? The white space looks huge with regards to the menu width.
I think either the menu should be bigger or the blank space smaller
![image](https://user-images.githubusercontent.com/31045223/221854883-f408683c-63cd-4ae8-9321-3ef671b71902.png)